generatebundlefile/Makefile

```
@@ -41,5 +41,9 @@ build: ## Build release binary.
 	go mod tidy -compat=1.17
 	go build -o bin/generatebundlefile *.go
 
+build-linux: lint fmt vet
+	mkdir bin
+	env GOOS=linux GOARCH=amd64 go build -v -o bin/generatebundlefile *.go
```

```suggestion
	env GOOS=linux GOARCH=amd64 $(MAKE) build
```
Don't repeat yourself. Plus if the build target changes, this will just work.